<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Поиск по:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="findComboBox">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>номеру клиента</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ФИО</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>паспорту</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="findLineEdit">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton#findButton {
     background-color: transparent;
	color:black;
 }
 QPushButton#findButton:pressed {
	background: transparent;
 }
 QPushButton#findButton:hover:!pressed{
	background: transparent;
	border: 1px solid black;
 }

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/img/find_but</normaloff>:/img/find_but</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>153</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#pushButton {
     background-color: transparent;
	color:black;
 }
 QPushButton#pushButton:pressed {
	color:#318ad4;
 }
 QPushButton#pushButton:hover:!pressed{
	color:	#0063cc;
	text-decoration: underline;
 }</string>
         </property>
         <property name="text">
          <string>Создайте</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>62</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>или</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>139</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#pushButton_2 {
     background-color: transparent;
	color:black;
 }
 QPushButton#pushButton_2:pressed {
	color:#318ad4;
 }
 QPushButton#pushButton_2:hover:!pressed{
	color:	#0063cc;
	text-decoration: underline;
 }</string>
         </property>
         <property name="text">
          <string>откройте</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>файл</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="make"/>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="saveAs"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="guide"/>
    <addaction name="aboutProgramm"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="make">
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="guide">
   <property name="text">
    <string>Руководство</string>
   </property>
  </action>
  <action name="aboutProgramm">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
